content/docs/start/data-management/metrics-parameters-plots.md

```diff
@@ -99,44 +99,39 @@ marked as a `metrics` file with `-M`. Its contents are:
 }
 ```
 
-> DVC doesn't force you to use any specific file names, nor does it enforce a
-> format or structure of a metrics file. It's completely user/case-defined.
-> Refer to `dvc metrics` for more details.
-
-You can view tracked metrics with DVC:
+You can view tracked metrics with `dvc metrics show `:
 
 ```dvc
 $ dvc metrics show
 Path                    avg_prec.test    avg_prec.train    roc_auc.test    roc_auc.train
 eval/live/metrics.json  0.94496          0.97723           0.96191         0.98737
 ```
 
-## Configuring plots
+[`evaluate.py`]:
+  https://github.com/iterative/example-get-started/blob/master/src/evaluate.py
+[roc-auc]:
+  https://scikit-learn.org/stable/modules/model_evaluation.html#receiver-operating-characteristic-roc
+[average precision]:
+  https://scikit-learn.org/stable/modules/model_evaluation.html#precision-recall-and-f-measures
+[metrics file]: /doc/command-reference/metrics#supported-file-formats
+
+The stage also writes [`roc_curve`] and [`confusion_matrix`] values in the
+`eval/live/plots` directory. Let's see how to visualize these better...
 
-`evaluate.py` also writes `precision`, `recall`, and `thresholds` arrays
-(obtained using
-[`precision_recall_curve`](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.precision_recall_curve.html))
-into the plots files `eval/prc/train.json` and `eval/prc/test.json`:
+## Rendering plots
+
+Similarly, `evaluate` writes `precision`, `recall`, and `thresholds` values into
+JSON arrays (shown below) in the the `eval/prc/train.json` and
+`eval/prc/test.json` [plots files]. It also generates a custom
+`eval/importance.png` image showing a bar chart of features' importance.
 
 ```json
-{
-  "prc": [
-    { "precision": 0.021473008227975116, "recall": 1.0, "threshold": 0.0 },
-    ...,
-    { "precision": 1.0, "recall": 0.009345794392523364, "threshold": 0.6 }
-  ]
-}
+{ "precision": 0.021473008227975116, "recall": 1.0, "threshold": 0.0 },
+...
```

Comment:
This is not the actual file structure, though. Why change it? I worry users could try to generate a file with that structure and be surprised it doesn't work as expected.

Reply:
I thought it was easier to read since we were referring to the JSON array above but you're right. Updated, PTAL.